packages/shared/package.json

```diff
@@ -35,6 +35,7 @@
         "@rollup/plugin-typescript": "^6.0.0",
         "@tsconfig/svelte": "^1.0.10",
         "@types/jest": "^27.0.1",
+        "@types/node": "^18.7.9",
```

Comment:
I suppose this is needed as a dep and not just a dev dependency?

Reply:
Nah, but we only need types for development so it should be a dev dependency.